packages/base/default-templates/isolated-and-edit.gts

```diff
@@ -12,22 +12,22 @@ export default class DefaultCardDefTemplate extends GlimmerComponent<{
     format: Format;
   };
 }> {
-  getFieldIcon = (key: string) => {
-    const field: Field<BaseDefConstructor> | undefined = getField(
-      this.args.model.constructor,
-      key,
+  private excludedFields = ['id', 'cardInfo'];
+
+  private get displayFields() {
+    let fields = Object.entries(this.args.fields).filter(
+      ([key]) => !this.excludedFields.includes(key),
     );
-    let fieldInstance = field?.card;
-    return fieldInstance?.icon;
-  };
+    return Object.fromEntries(fields);
+  }
+
   <template>
     <div class={{cn 'default-card-template' @format}}>
-      {{#each-in @fields as |key Field|}}
+      {{#each-in this.displayFields as |key Field|}}
```

Comment:
The cardInfo field is hidden currently as explained in description. However, here's the code if you want to see theming in action:
```
{{#if (eq @format 'edit')}}
        <FieldContainer
          @label='Card Info'
          @icon={{getFieldIcon @model 'cardInfo'}}
          data-test-field='cardInfo'
        >
          <@fields.cardInfo />
        </FieldContainer>
      {{/if}}
 ```
 
Currently only `CardContainer` and only its certain css variables are updated, so you'd only see the color and box-shadow changes. I have separately tested the variables by making changes to more components (cards-grid components, other base-realm default components, field-container and its related components, etc) and it works.